packages/lib-react-components/src/helpers/withLayer/withLayer.js

```diff
@@ -22,6 +23,8 @@ function withLayer (WrappedComponent) {
 
     return (
       <Layer
+        role='dialog'
+        aria-label={title}
```

Comment:
This `Layer` can probably be replaced with a HTML `<dialog>`, now that dialogs are supported in all browsers.
https://caniuse.com/dialog